README.md

```diff
@@ -4,14 +4,15 @@ A reference application for Node.js developers looking to learn more about using
 [Apache Cassandra][cassandra] and [DataStax Enterprise][dse] in their applications and 
 services. Learn more at [killrvideo.github.io][killrvideo].
 
-## Running Locally
+## Running Locally using Docker
 
-Use these guides to get started running KillrVideo locally on your development machine:
+Development should be pretty straight-forward if you are familiar with docker and docker-compose
```

Comment:
I think we have 2 cases here 1) for folks that only care to get things running quickly, and 2) for those that actually wish to perform some type of development work against the KillrVideo nodejs backend.  We also have to assume whoever is looking at this is not familiar with docker or docker-compose.  Maybe have the quick list of commands to execute to get going with Docker, but also leave the original links to "Getting Started with Node.js" for those who want to dig a little deeper and develop with their IDE.

Reply:
We have to add these docs links back when we will update documentation. These getting-started guides aren't valid for the updated NodeJS Backend (no-loopback, no setup scripts etc.)